Airports/EGNF/Runway.txt

```diff
@@ -0,0 +1,2 @@
+06 24  055 235 N053.18.57.450 W001.11.54.830 N053.19.03.600 W001.11.39.940
+18 36  174 354 N053.19.06.080 W001.11.44.750 N053.18.56.730 W001.11.43.330
```

Comment:
Noted: Rounded down not up, but irrelevant.

Reply:
makes zero difference as the ES generated centerlines draw it using the lat long of the thresholds